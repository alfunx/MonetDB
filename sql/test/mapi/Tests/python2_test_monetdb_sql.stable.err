stderr of test 'python2_test_monetdb_sql` in directory 'sql/test/mapi` itself:


# 11:54:47 >  
# 11:54:47 >   mserver5 "--config=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=39459 --set xrpc_port=48456 --set monet_prompt= --set mal_listing=2 --trace  "--dbname=mTests_src_test_mapi" --set mal_listing=0 ; echo ; echo Over..
# 11:54:47 >  

# builtin opt 	gdk_arch = 64bitx86_64-unknown-linux-gnu
# builtin opt 	gdk_version = 1.39.0
# builtin opt 	prefix = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert
# builtin opt 	exec_prefix = ${prefix}
# builtin opt 	gdk_dbname = demo
# builtin opt 	gdk_dbfarm = ${prefix}/var/MonetDB/dbfarm
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_alloc_map = no
# builtin opt 	gdk_vmtrim = yes
# builtin opt 	monet_admin = adm
# builtin opt 	monet_prompt = >
# builtin opt 	monet_welcome = yes
# builtin opt 	monet_mod_path = ${exec_prefix}/lib64/MonetDB
# builtin opt 	monet_daemon = no
# builtin opt 	host = localhost
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_clients = 2
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_debug = 0
# builtin opt 	standoff_ns = 
# builtin opt 	standoff_start = start
# builtin opt 	standoff_end = end
# config opt 	prefix = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert
# config opt 	config = ${prefix}/etc/monetdb5.conf
# config opt 	prefix = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert
# config opt 	exec_prefix = ${prefix}
# config opt 	gdk_dbfarm = ${prefix}/var/MonetDB5/dbfarm
# config opt 	monet_mod_path = ${exec_prefix}/lib64/MonetDB5:${exec_prefix}/lib64/MonetDB5/lib:${exec_prefix}/lib64/MonetDB5/bin
# config opt 	mero_pidfile = ${prefix}/var/run/MonetDB/merovingian.pid
# config opt 	mero_controlport = 50001
# config opt 	sql_optimizer = default_pipe
# config opt 	minimal_pipe = inline,remap,deadcode,multiplex,garbageCollector
# config opt 	default_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mitosis,mergetable,deadcode,commonTerms,joinPath,reorder,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	no_mitosis_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,commonTerms,joinPath,reorder,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	sequential_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,commonTerms,joinPath,reorder,deadcode,reduce,history,multiplex,garbageCollector
# config opt 	nov2009_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	replication_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,history,replication,multiplex,garbageCollector
# config opt 	accumulator_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,constants,commonTerms,joinPath,deadcode,reduce,accumulators,dataflow,history,multiplex,garbageCollector
# config opt 	recycler_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,deadcode,constants,commonTerms,joinPath,deadcode,recycle,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	cracker_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,selcrack,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	sidcrack_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,sidcrack,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	octopus_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mitosis,mergetable,deadcode,constants,commonTerms,joinPath,octopus,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	mapreduce_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mapreduce,mergetable,deadcode,commonTerms,joinPath,reorder,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	datacyclotron_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,datacyclotron,mergetable,deadcode,commonTerms,joinPath,reorder,deadcode,reduce,dataflow,history,replication,multiplex,garbageCollector
# config opt 	derive_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mitosis,mergetable,deadcode,commonTerms,derivePath,joinPath,reorder,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	dictionary_pipe = inline,remap,dictionary,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,history,multiplex,garbageCollector
# config opt 	compression_pipe = inline,remap,evaluate,costModel,coercions,emptySet,aliases,mergetable,deadcode,constants,commonTerms,joinPath,deadcode,reduce,dataflow,compression,dataflow,history,multiplex,garbageCollector
# config opt 	opt_debug = 
# cmdline opt 	config = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/etc/monetdb5.conf
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	monet_mod_path = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/bin
# cmdline opt 	gdk_dbfarm = /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/var/MonetDB5/dbfarm
# cmdline opt 	mapi_open = true
# cmdline opt 	xrpc_open = true
# cmdline opt 	mapi_port = 39459
# cmdline opt 	xrpc_port = 48456
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbname = mTests_src_test_mapi
# cmdline opt 	mal_listing = 0
#warning: please don't forget to set your vault key!
#(see /ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/etc/monetdb5.conf)

# 11:54:47 >  
# 11:54:47 >  ./python_test_monetdb_sql.SQL.sh python_test_monetdb_sql 
# 11:54:47 >  


# 11:54:47 >  
# 11:54:47 >  python /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/HG/default/source/sql/../python/test/runtests.py
# 11:54:47 >  

test_BLOB (__main__.Test_Capabilities) ... ok
test_BOOL (__main__.Test_Capabilities) ... ok
test_CHAR (__main__.Test_Capabilities) ... ok
test_DATE (__main__.Test_Capabilities) ... ok
test_DATETIME (__main__.Test_Capabilities) ... ok
test_DECIMAL (__main__.Test_Capabilities) ... ok
test_DOUBLE (__main__.Test_Capabilities) ... ok
test_INET (__main__.Test_Capabilities) ... ok
test_INT (__main__.Test_Capabilities) ... ok
test_REAL (__main__.Test_Capabilities) ... ok
test_TEXT (__main__.Test_Capabilities) ... ok
test_TIME (__main__.Test_Capabilities) ... ok
test_TIMESTAMP (__main__.Test_Capabilities) ... ok
test_TIMESTAMPTZ (__main__.Test_Capabilities) ... ok
test_TIMETZ (__main__.Test_Capabilities) ... ok
test_TINYINT (__main__.Test_Capabilities) ... ok
test_URL (__main__.Test_Capabilities) ... ok
test_bigresult (__main__.Test_Capabilities) ... ok
test_closecur (__main__.Test_Capabilities) ... ok
test_customtype (__main__.Test_Capabilities) ... ok
test_description (__main__.Test_Capabilities) ... ok
test_fractional_TIMESTAMP (__main__.Test_Capabilities) ... ok
test_multiple_queries (__main__.Test_Capabilities) ... ok
test_small_CHAR (__main__.Test_Capabilities) ... ok
test_transactions (__main__.Test_Capabilities) ... ok
test_truncation (__main__.Test_Capabilities) ... ok

----------------------------------------------------------------------
Ran 26 tests

OK
test_BINARY (__main__.Test_DBAPI20) ... ok
test_Binary (__main__.Test_DBAPI20) ... ok
test_DATETIME (__main__.Test_DBAPI20) ... ok
test_Date (__main__.Test_DBAPI20) ... ok
test_Exceptions (__main__.Test_DBAPI20) ... ok
test_ExceptionsAsConnectionAttributes (__main__.Test_DBAPI20) ... ok
test_NUMBER (__main__.Test_DBAPI20) ... ok
test_None (__main__.Test_DBAPI20) ... ok
test_ROWID (__main__.Test_DBAPI20) ... ok
test_STRING (__main__.Test_DBAPI20) ... ok
test_Time (__main__.Test_DBAPI20) ... ok
test_Timestamp (__main__.Test_DBAPI20) ... ok
test_apilevel (__main__.Test_DBAPI20) ... ok
test_arraysize (__main__.Test_DBAPI20) ... ok
test_callproc (__main__.Test_DBAPI20) ... ok
test_close (__main__.Test_DBAPI20) ... ok
test_commit (__main__.Test_DBAPI20) ... ok
test_connect (__main__.Test_DBAPI20) ... ok
test_cursor (__main__.Test_DBAPI20) ... ok
test_cursor_isolation (__main__.Test_DBAPI20) ... ok
test_description (__main__.Test_DBAPI20) ... ok
test_escape (__main__.Test_DBAPI20) ... ok
test_execute (__main__.Test_DBAPI20) ... ok
test_executemany (__main__.Test_DBAPI20) ... ok
test_fetchall (__main__.Test_DBAPI20) ... ok
test_fetchmany (__main__.Test_DBAPI20) ... ok
test_fetchone (__main__.Test_DBAPI20) ... ok
test_mixedfetch (__main__.Test_DBAPI20) ... ok
test_nextset (__main__.Test_DBAPI20) ... ok
test_paramstyle (__main__.Test_DBAPI20) ... ok
test_rollback (__main__.Test_DBAPI20) ... ok
test_rowcount (__main__.Test_DBAPI20) ... ok
test_setinputsizes (__main__.Test_DBAPI20) ... ok
test_setoutputsize_basic (__main__.Test_DBAPI20) ... ok
test_substring (__main__.Test_DBAPI20) ... ok
test_threadsafety (__main__.Test_DBAPI20) ... ok
test_unicode (__main__.Test_DBAPI20) ... ok
test_utf8 (__main__.Test_DBAPI20) ... ok

----------------------------------------------------------------------
Ran 38 tests

OK
test_Binary (test_pythonize.TestPythonize) ... ok

----------------------------------------------------------------------
Ran 1 test

OK
test_str_subclass (test_monetize.TestMonetize) ... ok
test_unknown_type (test_monetize.TestMonetize) ... ok

----------------------------------------------------------------------
Ran 2 tests

OK

# 11:54:48 >  
# 11:54:48 >  Done.
# 11:54:48 >  

