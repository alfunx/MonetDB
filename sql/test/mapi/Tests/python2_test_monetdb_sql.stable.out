stdout of test 'python2_test_monetdb_sql` in directory 'sql/test/mapi` itself:


# 11:54:47 >  
# 11:54:47 >   mserver5 "--config=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/etc/monetdb5.conf" --debug=10 --set gdk_nr_threads=0 --set "monet_mod_path=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/lib:/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/lib64/MonetDB5/bin" --set "gdk_dbfarm=/ufs/manegold/_/scratch0/Monet/HG/default/prefix.--enable-strict_--disable-debug_--enable-optimize_--enable-assert/var/MonetDB5/dbfarm"  --set mapi_open=true --set xrpc_open=true --set mapi_port=39459 --set xrpc_port=48456 --set monet_prompt= --set mal_listing=2 --trace  "--dbname=mTests_src_test_mapi" --set mal_listing=0 ; echo ; echo Over..
# 11:54:47 >  

# MonetDB server v5.21.0, based on kernel v1.39.0
# Not released
# Serving database 'mTests_src_test_mapi', using 4 threads
# Compiled for x86_64-unknown-linux-gnu/64bit with 64bit OIDs dynamically linked
# Found 7.751 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://monetdb.cwi.nl/ for further information
# Listening for connection requests on mapi:monetdb://rig.ins.cwi.nl:39459/
# MonetDB/SQL module v2.39.0 loaded
# MonetDB/GIS module v0.19.0 loaded

Ready.


# 11:54:47 >  
# 11:54:47 >  ./python_test_monetdb_sql.SQL.sh python_test_monetdb_sql 
# 11:54:47 >  


# 11:54:47 >  
# 11:54:47 >  python /net/rig.ins.cwi.nl/export/scratch0/manegold/Monet/HG/default/source/sql/../python/test/runtests.py
# 11:54:47 >  


# 11:54:48 >  
# 11:54:48 >  Done.
# 11:54:48 >  

